source/uwp/Renderer/lib/WholeItemsPanel.cpp

```diff
@@ -399,35 +399,48 @@ namespace AdaptiveNamespace
     void WholeItemsPanel::AddElementToStretchablesList(_In_ IUIElement* element)
     {
         ComPtr<IUIElement> localElement(element);
-        ComPtr<IUIElement4> elementWithAccessKey;
-        if (SUCCEEDED(localElement.As(&elementWithAccessKey)))
+        ComPtr<IFrameworkElement> elementAsFrameworkElement;
+        THROW_IF_FAILED(localElement.As(&elementAsFrameworkElement));
+
+        ComPtr<IInspectable> tagAsInspectable;
+        THROW_IF_FAILED(elementAsFrameworkElement->get_Tag(&tagAsInspectable));
+
+        ComPtr<IElementTagContent> elementTagContent;
+        if (tagAsInspectable != nullptr)
         {
-            std::string elementAccessKey = std::to_string(m_accessKeyCount);
-            ++m_accessKeyCount;
+            tagAsInspectable.As(&elementTagContent);
+            elementTagContent->put_IsStretchable(true);
+        }
+        else
+        {
+            ComPtr<IElementTagContent> tagContent;
+            THROW_IF_FAILED(MakeAndInitialize<ElementTagContent>(&tagContent, nullptr, nullptr, nullptr, nullptr, false, true));
```

Comment:
what are all these nullptr and booleans? Could we get a comment to make it more descriptive?

Reply:
They're non-relevant constructor parameters. But, this is silly. Added a constructor that doesn't take all this to allow callers to construct a default tag content object and make the code a little clearer.

---
In reply to: [389054235](https://github.com/microsoft/AdaptiveCards/pull/3820#discussion_r389054235) [](ancestors = 389054235)